elasticdl/python/ps/parameters.py

```diff
@@ -0,0 +1,49 @@
+import tensorflow as tf
+
+from elasticdl.python.common.ndarray import tensor_to_ndarray
+from elasticdl.python.ps.embedding_table import create_embedding_table
+
+
+class Parameters(object):
+    def __init__(self):
+        self.init_status = False
+        self.non_embedding_params = {}
+        self.embedding_params = {}
+
+    def get_embedding_param(self, name, indices):
+        if name not in self.embedding_params:
+            raise ValueError(
+                "Please initialize embedding param %s first!", name
+            )
+        return self.embedding_params[name].get(indices)
+
+    def set_embedding_param(self, name, indices, values):
+        if name not in self.embedding_params:
+            raise ValueError(
+                "Please initialize embedding param %s first!", name
+            )
+        self.embedding_params[name].set(indices, values)
+
+    def init_from_model_pb(self, model_pb):
```

Comment:
Would it be better to assume that interface `init_from_model_pb` takes a tensor list and a EmbeddingInfo list? If so, we can put the logic of parsing protocol buffer into RPC servicer.

Reply:
I think that `Parameters` is actually an in-memory data structure of `proto::Model`.  We could deserialize from a model_pb, and create a parameters object. We could also serialize a parameters object directly to a model_pb file, for the next step serving in C++. 

So it makes sense that we have an interface which could init a parameters object directly from model_pb.